modules/jaggery-apps/user-dashboard/dashboard/index.jag

```diff
@@ -3,27 +3,22 @@
 
         var user = session.get('user');
         var log = new Log();
-        var util = require("util/utility.jag");
         var authzFailure = request.getParameter("authz_failure");
-
-        util.setServerURL("true");
+        var identityServerURL = Packages.org.wso2.carbon.identity.core.util.IdentityUtil;
 
         if (user == null) {
-        if (authzFailure != null) {
-        response.sendRedirect("login.jag?authz_failure=" + authzFailure);
-        } else {
-        response.sendRedirect("login.jag");
-        }
+            if (authzFailure != null) {
+                response.sendRedirect("login.jag?authz_failure=" + authzFailure);
+            } else {
+                response.sendRedirect("login.jag");
+            }
         }
 
-        var httpsCoreUrl = application.get(SERVER_CORE_URL);
-        if (httpsCoreUrl === 'null' || httpsCoreUrl == null || httpsCoreUrl.length <= 0) {
-        httpsCoreUrl = '';
-        }
+        var httpsCoreUrl = identityServerURL.getServerURL("", true, true);
```

Comment:
Why read this back again from the util class. Shouldn't the SERVER_CORE_URL being populated should be used here